src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 0) {
+    return 0;
+  }
+
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7) {
+    return days * 40 - 20;
+  }
+
+  return days * 40 - 50;
```

Comment:
move all 'magic' numebrs to the variables

Reply:
fixed